gatsby-browser.js

```diff
@@ -23,6 +23,10 @@ const GlobalStyles = createGlobalStyle`
         background-color: ${prop => prop.theme.colors.main1};
     }
 
+    body:has(#containerDiv) {
```

Comment:
Why? 😭

Reply:
jajaj I was trying to do something, but, it didn't work. I forgot to delete it. I'm sorry.